ikarus/finiteelements/mechanics/enhancedassumedstrains.hh

```diff
@@ -608,8 +300,8 @@ protected:
   }
 
 private:
-  using EAS2DVariantImpl = EAS2DVariant<typename LocalView::Element::Geometry>;
-  using EAS3DVariantImpl = EAS3DVariant<typename LocalView::Element::Geometry>;
+  using EAS2DVariantImpl = EAS::Variants<Geometry>::EAS2D;
```

Comment:
This is a not needed indirection. Can you refactor the struct in `easvariants.hh` such that here `
 ```cpp
using EASVariant = EAS::Variants<Geometry>::type;
 ```
 is left. You can use `std::conditional_t` and decide which variant you use with the dimension of the geometry.